### PR TITLE
Check user status if is suspended and throw exception

### PR DESCRIPTION
**Pull request by lionick**

### CODE REVIEW

File: lib/Auth/Process/COmanageDbClient.php [hunk 1]
```diff
@@ -241,6 +241,9 @@ public function process(&$state)
                 $state['basicInfo'] = $basicInfo;
             }
             if (empty($basicInfo['id']) || empty($basicInfo['status']) || ($basicInfo['status'] !== 'A' && $basicInfo['status'] !== 'GP')) {
+                  if ($basicInfo['status'] === 'S') {
+                      throw new SimpleSAML_Error_Exception('You account is suspended. Please contact support for further assistance.');
```
Comment (NicolasLiampotis) [inline lib/Auth/Process/COmanageDbClient.php:245]: minor edit "Your account has been suspended"

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

**Comment by ioigoume** [comment]
LGTM

File: lib/Auth/Process/COmanageDbClient.php (hunk 1 above)
Comment (lionick) [reply lib/Auth/Process/COmanageDbClient.php]: I have made the change.

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**